### ISSUE TITLE
An issue about preprocessing

### ISSUE THREAD

**Issue by QuXing9** [WSP-LAB/Montage#4]
## Description

Hi, When running the montage on my machine, there are some problems that I can't solve. Hoping to get your help, Thx!

When I use chakra to preprocess the dataset that you provide at the [link](https://github.com/WSP-LAB/js-test-suite). There are many errors written in the log files in the `data\log` directory. The error messages are as follows:

```
============== STDOUT ===============

============== STDERR ===============
Couldn't load file 'fcfbc86708bc3a4062c2091a062e13b6.js'
Couldn't load file 'faa81dc98fc13338aca921d45eebae79.js'  
```

So,  the problem is that chakra can't load the js files with the `WScript.LoadScriptFile` function correctly. And I tried to enter the directory `js-test-suite/testsuite` , then execute the written files with chakra, it reported the same error.  I don't know it is caused by `WScript.LoadScriptFile`, or a bug of chakra engine, or what parameter I'm missing. Is there a solution to this problem? Looking forward to your reply! Thanks!

## Environment

chakra version: 1.13.0 (release)

Ubuntu: 18.04

**Comment by leeswimming** [comment]
Since v1.8.0, it seems that ChakraCore has changed how to load scripts when relative paths are passed to `WScript.LoadScriptFile`. The straightforward solution for this problem is rewriting seed files to provide absolute paths to `WScript.LoadScriptFile`. For instance, replace `load('fcfbc86708bc3a4062c2091a062e13b6.js');` in the seed file with `load('/home/user/js-test-suite/testsuite/fcfbc86708bc3a4062c2091a062e13b6.js');`.

**Comment by leeswimming** [comment]
Fixed in 6e5859b. I fixed the wrapper function to automatically prepend the seed directory path.

**Comment by QuXing9** [comment]
> Fixed in [6e5859b](https://github.com/WSP-LAB/Montage/commit/6e5859bc7add83107d84dd849bfa3d4959a86dcc). I fixed the wrapper function to automatically prepend the seed directory path.

Got it! Thanks for your prompt reply.
